pom.xml

```diff
@@ -21,97 +26,104 @@
             <artifactId>model</artifactId>
             <version>${project.version}</version>
         </dependency>
-        
+
         <dependency>
             <groupId>io.streamzi.cloudevent-flow</groupId>
             <artifactId>container</artifactId>
             <version>${project.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>io.streamzi</groupId>
+            <artifactId>jcloudevents-api</artifactId>
+            <version>0.0.1-SNAPSHOT</version>
```

Comment:
I think I've introduced the proper 0.0.2 version in my `cloudevent-flow` change. since this has that as dependency IMO this is not needed